src/components/Chat/AssistantSubModelSelector.tsx

```diff
@@ -0,0 +1,44 @@
+import { useMemo } from 'react';
+
+import { useTranslation } from 'next-i18next';
+
+import { OpenAIEntityModel } from '@/src/types/openai';
+
+import { useAppSelector } from '@/src/store/hooks';
+import { ModelsSelectors } from '@/src/store/models/models.reducers';
+
+import { Combobox } from '../Common/Combobox';
+import { ModelSelectRow, SettingContainer } from './ConversationSettings';
+
+interface Props {
+  assistantModelId: string;
+  onSelectAssistantSubModel: (modelId: string) => void;
+}
+
+export const AssistantSubModelSelector = ({
+  assistantModelId,
+  onSelectAssistantSubModel,
+}: Props) => {
+  const { t } = useTranslation('chat');
+  const modelsModels = useAppSelector(ModelsSelectors.selectModelsOnly);
+  const modelsMap = useAppSelector(ModelsSelectors.selectModelsMap);
+  const assistantSubModel = useMemo(
+    () => modelsMap[assistantModelId],
+    [assistantModelId, modelsMap],
+  );
+  return (
+    <SettingContainer>
```

Comment:
Let's move out this setting container as you did for another settings and use empty fragment here instead <></>

Reply:
Changes added as suggested.